README.md

```diff
@@ -1,7 +1,12 @@
-# Plantilla del curso de desarrollo ágil 
+# Sistema de Gestión para Taller de Coches
 
-Plantilla para el [curso de desarrollo ágil](https://jj.github.io/curso-tdd)
+## Descripción del Problema
+Un taller de coches con varias sedes en distintas ciudades de España está enfrentando problemas para gestionar de manera eficiente la distribución y disponibilidad de las piezas necesarias para las reparaciones. El taller necesita un sistema que no solo identifique qué sede tiene la pieza requerida, sino que también optimice el proceso considerando factores como:
+Costos de transporte entre sedes,
+tiempos de entrega estimados de las piezas,
+priorización de las reparaciones según la urgencia del cliente y la disponibilidad de las piezas.
 
-Contiene el mínimo para un proyecto, incluyendo una GitHub action que
-comprueba la presencia de ciertos ficheros, un directorio para los
-tests, y otro para la documentación.
+El sistema debería ser capaz de predecir las futuras necesidades de piezas en función de patrones de demanda, evitando que quedar sin stock cuando más lo necesita. Esto permitirá al taller reducir retrasos y optimizar sus recursos, mejorando la satisfacción del cliente y la productividad general.
+
+
+https://github.com/lmchaves/OrganizarTaller/pull/1
```

Comment:
¿Qué necesidad cubre esto?
También, ¿crees que quitar la palabra "también" de una descripción realmente elimina el hecho de que estás describiendo varios "problemas" (en realidad no, es una solución, no un problema) en vez de uno solo? Lo que has hecho aquí: 90c0077d322f46de62fc0044a453f4f340907485
Se trata de que resolváis realmente los comentarios, no simplemente que lo eliminéis. Hacer lo que has hecho simplemente genera más trabajo por las dos partes.